features/step_definitions/metering.rb

```diff
@@ -261,10 +265,27 @@
 # XXX: currently OLM uninstall is TBD, we uninstall by removing the namespace
 Given /^the#{OPT_QUOTED} metering service is uninstalled using OLM$/ do | metering_ns |
   ensure_admin_tagged
+  ensure_destructive_tagged
+
   metering_ns ||= "openshift-metering"
   step %Q/I switch to cluster admin pseudo user/
   project("openshift-marketplace")
   step %Q/I ensure "metering" catalogsourceconfig is deleted/
   project(metering_ns)
   step %Q/I ensure "#{metering_ns}" project is deleted/
 end
+
+Given /^all datasources for#{OPT_QUOTED} project are imported$/ do | proj_name |
```

Comment:
maybe `report datasources` will speak better

Reply:
the alias that we call from `oc` is `datasources`...even though the YAML report `ReportDataSources`.  Perhaps I can talk with Chance about that.